DuckDuckGo/Configuration/ConfigurationStore.swift

```diff
@@ -34,30 +35,94 @@ final class ConfigurationStore: ConfigurationStoring {
         .remoteMessagingConfig: "remote-messaging-config.json"
     ]
 
-    static let shared = ConfigurationStore()
+    private enum Constants {
+        static let configStorageTrackerRadarEtag = "config.storage.trackerradar.etag"
+        static let configStorageBloomFilterSpecEtag = "config.storage.bloomfilter.spec.etag"
+        static let configStorageBloomFilterBinaryEtag = "config.storage.bloomfilter.binary.etag"
+        static let configStorageBloomFilterExclusionsEtag = "config.storage.bloomfilter.exclusions.etag"
+        static let configStorageSurrogatesEtag = "config.storage.surrogates.etag"
+        static let configStoragePrivacyConfigurationEtag = "config.storage.privacyconfiguration.etag"
+        static let configFBConfigEtag = "config.storage.fbconfig.etag"
```

Comment:
We can remove all the FBConfig references, those have recently been removed (cc @ladamski)